### DISCUSSION TITLE
VBET Validation Punchlist of Issues Noticed

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#329] [VBET] [answered]
Just running log of notes and reminders.
- Noticed in [Kansas Bear 11040005](https://data.riverscapes.xyz/#/PUBLIC_DATA/5c8e0efb-26f6-432c-8b80-ae702d7a3bde) that this is one of the best "low relief" test beds to play with. 
- [ ] In  [Kansas Bear 11040005](https://data.riverscapes.xyz/#/PUBLIC_DATA/5c8e0efb-26f6-432c-8b80-ae702d7a3bde) extent of perennial network leaves most of basin unanalyzed, but including intermittent looks good. Note that perennial is really discontinuous too. Maybe in these [arid ecoregions](https://gist.github.com/lauren-herbine/a6d3b8d0ee23e07e702d522762127ed7) we make sure to include intermittent as primary view? 
- [ ] Surprisingly the extreme "low relief" of the [Bayou Teche 08080102](https://data.riverscapes.xyz/#/PUBLIC_DATA/774c9d2a-064d-4990-8531-2f792881d24b) worked really well on the western edge even in headwaters for easy valley bottom delineation. We will need to pay attention to the broad plains to the west where the HUC is bounded by the broad plains of the Red River and has a 10-15 km wide river right valley bottom! Many of these lower basin HUCs have a boundary that is only half or one side of the drainage wing. Also really cool is that the 10 m DEM is good enough to see all the natural levee systems built up on this valley bottom.
- [ ] For the mainstem Mississippi [(e.g. Lower Mississippi-Memphis 08010100](https://data.riverscapes.xyz/#/PUBLIC_DATA/3e1fb062-c055-45cb-9f11-7cfb08486237) the active valley bottom is impressive (e.g. 5 to 10 km in width), with extremely obvious scroll bars and ridge and swale topography indicative of active lateral accreting floodplains. However, we'll need to pay attention to the **inconsistently bounded western edge** (not sure if it is ever against a valley bottom margin) and only portions of the eastern edge are properly bounded by uplands. Generically, what do we do for HUCs where valley bottom is not bounded on one side? Maybe not a problem?
- [ ] The [Flambeau 07050002](https://data.riverscapes.xyz/#/PUBLIC_DATA/bdb51760-8543-449c-bdaa-8e356b6ed45f) is a fantastic example of glacial plains and the land of lakes with expansive wetlands, fens, ponds and lakes acting as headwaters. This low relief environment was surprisingly easy to read the valley bottoms for. However, this watershed really highlights the problem of what to do for valley bottom mapping of water bodies (both natural and artificial)?  Need a strategy.
- [x] We need to decide what the default inputs from NHD+HR are. Previously it has been 46006 (perennial), 55800 (Artificial Path), 33400 (Connectors). We think we should derive LOEs and valley bottoms for those plus 46003 (intermittent) and 46007 (ephemeral), but turn these later two off by default in display and report metrics separately for them.
- [x] We need to decide if we want to use D8 or tauDEM for slope analysis. How costly is it to produce this?
- [x] We need to decide what we want for the likelihood scale (0 - .99?)
- [x] What do we think about the existing categories in VBET? Keep them or make some changes?

**Comment by lauren-herbine** [comment]
Adding some of my observations (will continually update):

- [ ] Echoing Joe's observation of the [Flambeau](https://data.riverscapes.xyz/#/PUBLIC_DATA/bdb51760-8543-449c-bdaa-8e356b6ed45f), these series of lakes are labeled as part of the perennial network (ex below). "active channel" will be far off from the line network. 
![image](https://user-images.githubusercontent.com/73319684/124642078-a3af7d80-de4c-11eb-9595-faecd2bd1431.png)

- [ ] Likely decisions made about fens/bogs/meadows (low relief, no channel, all floodplain areas) will be able to be applied in very diverse regions. The aforementioned glacial plains, some Appalachian plateaus, and the delta of the mainstem of the Mississippi.
![image](https://user-images.githubusercontent.com/73319684/124645407-a0b68c00-de50-11eb-991b-bd321f4e4b53.png)

- [ ] For 55800 (artificial paths) and 33400 (connectors) we need to decide what we're doing with natural waterbodies, as they are the flowlines that appear crossing/connecting in/flowing along NHDWaterbodies. 
- [ ] On a related note to above, there many instances where river areas (polygons) fall under "River Polygons," which are part of the NHDFlowlines. We have already decided to use the polygons in these situations. However, there are also some places (again, the glaciated plains are where we see a lot of this) where the river areas fall under "Lakes, Ponds, Reservoirs and Waterbodies," which is a part of NHDWaterbodies. We have decided to use these polygons instead of buffering the flowlines in these instances. See #331 
![image](https://user-images.githubusercontent.com/73319684/124662548-3d375900-de66-11eb-8975-9a0998035f03.png)

**Comment by shelbysawyer** [comment]
- [x] What attribute is being us to segment the valley bottom sausage links?
- [x] Decide on selection query.. group by drainage area bins, users, etc?

**Comment by joewheaton** [comment, marked as answer]
These issues have been addressed in subsequent versions of VBET.
